lib/ProductOpener/Images.pm

```diff
@@ -357,6 +358,23 @@ sub scan_code ($file) {
 	return $code;
 }
 
+sub get_selected_image_uploader ($product_ref, $imagefield) {
+
+	# Retrieve the product's image data
+	my $image_ref = $product_ref->{images}{$imagefield};
+
+	if ($image_ref && $image_ref->{imgid}) {
```

Comment:
This will fail if $image_ref is not defined (if we try to get the uploader of a selected image that does not exist).

Instead we can do something like $imgid = deep_get($product_ref, "images", $imagefield, "imgid");

Then we can use deep_get as well to get the uploader